docs/content/direct/contribute.md

```diff
@@ -1,7 +1,7 @@
 # Contributing to KubeStellar
 Greetings! We are grateful for your interest in joining the KubeStellar community and making a positive impact. Whether you're raising issues, enhancing documentation, fixing bugs, or developing new features, your contributions are essential to our success.
 
-To get started, kindly read through familiarize yourself with our [code of conduct](../Contribution guidelines/coc.md).
+To get started, kindly read through and familiarize yourself with our [Code of Conduct](https://github.com/kubestellar/kubestellar/blob/main/CODE_OF_CONDUCT.md).
```

Comment:
This should be a relative link to a page that appears in the navigation panel on the left side of the website (when the browser is opened wide enough).
- Relative, so that navigation stays within one version of the website.
- To a page that appears in the navigation panel, so that the reader does not find herself in nowhere-land once she follows the link.

<img width="896" alt="image" src="https://github.com/user-attachments/assets/1dc6a1cc-9554-45be-ac63-c507b3b2cf32" />